tests/suites/test_suite_ssl.function

```diff
@@ -2073,7 +2077,7 @@ void perform_handshake( handshake_test_options* options )
     }
 #endif
 
-    if( options->expected_negotiated_version == TEST_SSL_MINOR_VERSION_NONE )
+    if( (int)options->expected_negotiated_version == TEST_SSL_MINOR_VERSION_NONE )
```

Comment:
Not possible to use MBEDTLS_SSL_VERSION_UNKNOWN instead of TEST_SSL_MINOR_VERSION_NONE ?

Reply:
:rocket: